Source/CarthageKit/GitHub.swift

```diff
@@ -265,6 +268,28 @@ private func fetchAllPages(URL: NSURL, credentials: GitHubCredentials?) -> ColdS
 			let thisData = ColdSignal.single(data)
 
 			if let HTTPResponse = response as? NSHTTPURLResponse {
+				if HTTPResponse.statusCode != 200 && HTTPResponse.statusCode != 404 {
```

Comment:
Maybe this should just check for codes in the 4xx and 5xx blocks? 2xx and 3xx aren't really failures AFAIK.